samples/subsys/display/lvgl/src/main.c

```diff
@@ -36,12 +33,26 @@ static void button_isr_callback(const struct device *port,
 }
 #endif
 
+#if IS_ENABLED(CONFIG_LV_Z_POINTER_KSCAN) || IS_ENABLED(CONFIG_LV_Z_POINTER_INPUT)
+static void button_click_callback(lv_event_t *e)
+{
+	lv_event_code_t code = lv_event_get_code(e);
+
+	if (code == LV_EVENT_CLICKED) {
```

Comment:
Nit: Since you only add the cb for `LV_EVENT_CLICK` this is not needed but if you want to keep it I'd move the `lv_event_get_code` in into the check.

Reply:
Also: Can you maybe amend the first commit to have the callback code correctly formatted from the start and only deal with the existing code in this commit?

Reply:
fixed